.gitignore

```diff
@@ -0,0 +1 @@
+test.md 
```

Comment:
- [ ] You should add `node_modules` in your `.gitignote` file. Add that part in the next line. Example:
```
test.md
node_modules
```